### PR TITLE
define .md extension as going through jupytext as well

### PR DESCRIPTION
**Pull request by parmentelat**
this change lets me manage notebooks stored by jupyter as md:myst

I understand it's probably too intrusive as such, but I'd love to achieve this effect, either in a rough way like this, or through some configuration

what do you think ?

### CODE REVIEW

File: ipypublish/convert/main.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def _default_pre_conversion_funcs(self):
             # this is deprecated in newer versions
             from jupytext import readf as read  # noqa: F401
 
-        return {".Rmd": read}
+        return {".Rmd": read, ".md": read, }
```
Comment (chrisjsewell) [inline ipypublish/convert/main.py:135]:
```suggestion
        return {".Rmd": read, ".md": read}
```

**Comment by chrisjsewell** [comment]
thanks! yeh I don;t see an issue with this, note you still have to set the source suffix in your conf.py if using it with sphinx:
https://ipypublish--134.org.readthedocs.build/en/134/sphinx_ext_notebook.html#usage

**Comment by parmentelat** [comment]
Many thanks, this is really helpful  

Any chance to have this released ?  
I understand your plans about ipypublish are to spend as little time as possible, so maybe all the more reason to publish very small changes  
in any case it'd save me a lot of headaches if that could be part of the official release by beg. of September when the teaching season reboots ;)

cheers

**Comment by chrisjsewell** [comment]
> Any chance to have this released ?

Done 👍

**Comment by parmentelat** [comment]
one million thanks !

**Pull request merged**